specificatie/openapi.yaml

```diff
@@ -186,16 +185,16 @@ paths:
           required: false
           schema:
             type: string
-            maxLength: 16
+            pattern: "^[0-9]{16}$"
             example: "0518200000366054"
         - in: query
           name: verblijfplaats__straat
           description: |
-                        Een naam die door de gemeente aan een openbare ruimte is gegeven. **Gebruik van de wildcard is toegestaan. Zie [feature-beschrijving](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.2.0/features/wildcard.feature)** **Zoeken met tekstvelden is [case-Insensitive](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.1.0/features/case_insensitive.feature).
+                        Een naam die door de gemeente aan een openbare ruimte is gegeven. **Gebruik van de wildcard is toegestaan. Zie [feature-beschrijving](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/wildcard.feature)** **Zoeken met tekstvelden is [case-Insensitive](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.1.0/features/case_insensitive.feature).
           required: false
           schema:
             type: string
-            maxLength: 80
+            pattern: ^(\?+|\*)?[a-zA-Z0-9À-ÿ' '\-\']{1,80}(\?+|\*)?$
```

Comment:
niet gelijk aan de pattern in de request body. De ' voor en achter de spatie kan weg.

Reply:
Aangepast